src/sentry/migrations/0904_onboarding_task_project_id_idx.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.2.1 on 2025-05-19 22:18
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.new_migrations.monkey.special import SafeRunSQL
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = True
+
+    dependencies = [
+        ("sentry", "0903_missing_indexes_in_state"),
+    ]
+
+    operations = [
+        SafeRunSQL(
+            'CREATE INDEX CONCURRENTLY "sentry_organizationonboardingtask_project_id_bba0eae8" ON "sentry_organizationonboardingtask" ("project_id");',
```

Comment:
this is what django thinks it should be called (forced by removing and re-adding `db_index=...`)